src/app/api.js

```diff
@@ -53,6 +53,14 @@ const wangBase = {
   ].map(makeLayerNameToConfig('wang')),
 };
 
+const vanderbiltDescription = 'High Bit Depth (uint16) Multiplex immunofluorescence Imaging';
```

Comment:
```suggestion
const vanderbiltDescription = 'High Bit Depth (uint16) Multiplex Immunofluorescence Imaging';
```

Reply:
Looks like I still can't get that right...